index.js

```diff
@@ -313,6 +315,17 @@ function Argv (processArgs, cwd) {
     return options
   }
 
+  var groups = {'Options:': []}
+  self.group = function (opts, groupName) {
+    if (!Array.isArray(opts)) opts = [opts]
+    if (!groups[groupName]) groups[groupName] = []
+    Array.prototype.push.apply(groups[groupName], opts)
```

Comment:
https://jsperf.com/array-prototype-push-apply-vs-concat/13 note that using `apply` is significantly slower, despite the extra assignment used by `concat` interesting.